manifests/a/aloneguid/BrowserTamer/2.5.4/aloneguid.BrowserTamer.installer.yaml

```diff
@@ -0,0 +1,21 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.7-2-3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: aloneguid.BrowserTamer
+PackageVersion: 2.5.4
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Scope: user
+UpgradeBehavior: install
+Protocols:
+- http
+- https
+- x-bt
+ReleaseDate: 2022-05-12
+Installers:
+- Architecture: x64
+  InstallerUrl: https://www.aloneguid.uk/projects/bt/bin/bt-installer-2.5.4.exe
+  InstallerSha256: 57E9DBFCE1F4B520F7B85D7BEED077F15E56B9AE19CEE98E1636E54BE69650B8
+  ProductCode: Browser Tamer
```

Comment:
Might be good to include AppsAndFeatures entries since the version and publisher are unknown so that ARP matching can be better once MSFT implements it
```yaml
  AppsAndFeaturesEntries: 
  - DisplayName:  Browser Tamer 2.5.4
    DisplayVersion: 
    Publisher: 
    ProductCode: Browser Tamer
```